packages/journalforing/src/components/PDFVisning.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { FunctionComponent, useState } from 'react';
+import { Document, Page, pdfjs } from 'react-pdf';
+import { Loader, Pagination } from '@navikt/ds-react';
+import { FormattedMessage } from 'react-intl';
+import styles from './pdfvisning.less';
+import JournalDokument from '../typer/journalDokumentTsType';
+
+// eslint-disable-next-line
+const pdfjsWorker = require('pdfjs-dist/build/pdf.worker.entry');
```

Comment:
bruk heller import. Sjekk om dette funkar (Gjorde eit raskt søk)

import * as pdfjsLib from 'pdfjs-dist';
pdfjsLib.GlobalWorkerOptions.workerSrc = '//mozilla.github.io/pdf.js/build/pdf.worker.js';

Reply:
Fikk ikke dette til å fungere. Men vi har ikke helt bestemt oss for hvordan PDF visning skal være enda, så lar det være slik du foreslår og ser nærmere på det når vi bestemmer oss.